src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -0,0 +1,137 @@
+import logging
+import os
+import sas
+import sas_testcase
+
+from util import winnforum_testcase, getRandomLatLongInPolygon, makePpaAndPalRecordsConsistent, QueryPropagationAntennaModel
+
+def cbsddata(device):
+    installationParam = device['installationParam']
+    if 'antennaAzimuth' not in installationParam:
+        installationParam['antennaAzimuth'] = None
+        
+    cbsd = {'latitude': installationParam['latitude'],
+             'longitude': installationParam['longitude'],
+              'height': installationParam['height'],
+              'heightType': installationParam['heightType'],
+              'indoorDeployment': installationParam['indoorDeployment'],
+              'antennaAzimuth': installationParam['antennaAzimuth'],
+              'antennaGain': installationParam['antennaGain'],
+              'antennaBeamwidth': installationParam['antennaBeamwidth'],
+              'cbsdCategory': device['cbsdCategory']}
+    return cbsd
+
+def fssdata(fss_record, rxAntennaGainRequired=None):
+    deploymentParam = fss_record['deploymentParam'][0]
+    installationParam = deploymentParam['installationParam']
+    if 'antennaAzimuth' not in installationParam:
+        installationParam['antennaAzimuth'] = ''
+        
+    fss = {'latitude': installationParam['latitude'],
+             'longitude': installationParam['longitude'],
+              'height': installationParam['height'],
+              'antennaAzimuth': installationParam['antennaAzimuth'],
+              'antennaGain': installationParam['antennaGain'],
+              'antennaDowntilt': installationParam['antennaDowntilt']}
+    if rxAntennaGainRequired is not None:
+        fss['rxAntennaGainRequired'] = rxAntennaGainRequired
+    return fss
+
+
+class PropAndAntennaModelTestcase(sas_testcase.SasTestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    #self._sas_admin.Reset()
+
+  def tearDown(self):
+      pass
+
+  def generate_FT_S_PAT_default_config(self, testcasedir):
+
+# Load Devices
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+   
+    # load FSS
+    fss_record_0 = json.load(
+      open(os.path.join('testcases', 'testdata', 'fss_record_0.json')))
+    fss_record_0['rxAntennaGainRequired'] = True
+    reliabilityLevel = -1
+    with open(os.path.join(testcasedir, 'default_fss.json'), "w") as outfile:
+        json.dump( {'reliabilityLevel': reliabilityLevel, 'cbsd': cbsddata(device_a),'fss': fssdata(fss_record_0, True)}, outfile, indent=4, separators=(',', ': '))
+       
+    # Load PPA
+    ppa_record = json.load(
+        open(os.path.join('testcases', 'testdata', 'ppa_record_3.json')))
+     
+    reliabilityLevel = -1
+    with open(os.path.join(testcasedir, 'default_ppa.json'), "w") as outfile:
+        json.dump({'reliabilityLevel': reliabilityLevel, 'cbsd': cbsddata(device_a),'ppa': ppa_record['zone']['features'][0]}, outfile, indent=4, separators=(',', ': '))
```

Comment:
1) Please use the writeConfig() helper function.
2) Please format the text so that it's easier to read (see the config generation step of e.g. REG.11 for reference).
3) Please take a list of (cbsd, entity, etc.) tuples so that we can run multiple prop model checks within one test case (thus enabling us to check the pass percentage).